cmd/dao_tests/pubkey.go

```diff
@@ -0,0 +1,208 @@
+package main
+
+import (
+	"context"
+	"testing"
+
+	"github.com/RHEnVision/provisioning-backend/internal/dao"
+	_ "github.com/RHEnVision/provisioning-backend/internal/dao/sqlx"
+	"github.com/RHEnVision/provisioning-backend/internal/models"
+	"github.com/stretchr/testify/assert"
+
+	"github.com/RHEnVision/provisioning-backend/internal/config"
+	"github.com/RHEnVision/provisioning-backend/internal/db"
+)
+
+func createPk() []*models.Pubkey {
```

Comment:
This is used in a single test, having it a function seems not necessary and might confuse ppl who are adding tests, that they are supposed to use it, is there a reason this is not a local variable in the test that needs it?